app/Http/Controllers/GroupController.php

```diff
@@ -49,7 +50,7 @@ public function store(Request $request)
             // restaurant/member_id will be array later
             'restaurant_id' =>  'required|min:1|max:30',
             'member_id'     =>  'required|min:1|max:30',
-            'image'         =>  'required|mimes:jpeg,jpg,png,gif|max:1048'
+            'image'         =>  'mimes:jpeg,jpg,png,gif|max:2048'
```

Comment:
for image, I suggest to upload the image in a separate directory instead of saving the entire image in the db.
we will just save the path of the image like `/storage/images/filename.jpg`

Reply:
I don't think I understand what you mean. I have created a file called images inside a storage directory, but I have no idea what to do next.

Reply:
Reference:
// app/Http/Controllers/UploadController.php

namespace App\Http\Controllers;

use Illuminate\Http\Request;
use App\Upload;

class UploadController extends Controller
{
    public function showForm()
    {
        return view('upload.form');
    }

    public function upload(Request $request)
    {
        $request->validate([
            'file' => 'required|file|mimes:jpeg,png,pdf|max:2048',
        ]);

        $file = $request->file('file');
        $filename = time() . '_' . $file->getClientOriginalName();
        $file->move(public_path('uploads'), $filename);

        $upload = Upload::create(['filename' => $filename]);

        return redirect()->route('upload.form')->with('success', 'File uploaded successfully!');
    }
}

Reply:
Reference:
 public function upload(Request $request)
    {
        $request->validate([
            'file' => 'required|file|mimes:jpeg,png,pdf|max:2048',
        ]);

        $file = $request->file('file');
        $filename = time() . '_' . $file->getClientOriginalName();
        $file->move(public_path('uploads'), $filename);

        $upload = Upload::create(['filename' => $filename]);

        return redirect()->route('upload.form')->with('success', 'File uploaded successfully!');
    }